src/pages/LandingPage/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { LoginButton } from '../../components/LoginButton';
+import { CTA_RegisterButton } from '../../components/RegisterButton';
```

Comment:
What CTA does mean? 😋

Reply:
Call To Action xd the button of the landing page which is there to captivate the attention of the user and lead them to make an action xd, in this case, register :v